modules/gapi/test/serialization/gsrlzkernel.hpp

```diff
@@ -0,0 +1,376 @@
+// This file is part of OpenCV project.
```

Comment:
Please mark this file for removal, as stated in the comment for modules/gapi/test/serialization/gsrlzkernel.cpp

Reply:
???

Reply:
Ok, explaining here. The proper way to test the serialization is to test every entry we serialize/deserialize individually.

The top-level API and a usage model is again different (I put it to a document last week).

Once we start testing the serialization via that _final_ APIs, this synthetic backend makes no sense